scripts/cl_ui.lua

```diff
@@ -10,9 +10,13 @@ function openWarning(author, reason, tTitle, tWarnedBy, tInstructions)
     Citizen.CreateThread(function()
         local countLimit = 100 --10 seconds
         local count = 0
+        local SpaceKey = 22
+        if GetConvar("gamename", "rdr3") then 
```

Comment:
Perhaps implement something like what the default `chat` resource has for checking if the current environment is RDR2 or not:

https://github.com/citizenfx/cfx-server-data/blob/master/resources/%5Bgameplay%5D/chat/cl_chat.lua#L1

.. also, this condition will always return a truthy value. You probably wanted `GetConvar("gamename", "gta5") == "rdr2"`, but yea, I'd go with `chat`'s way of checking.

Reply:
that was good, thanks